pkg/config/draftconfig_template_test.go

```diff
@@ -3,13 +3,17 @@ package config
 import (
 	"fmt"
 	"io/fs"
+	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/Azure/draft/template"
+	"github.com/blang/semver/v4"
 	"github.com/stretchr/testify/assert"
 )
```

Comment:
question: why is the file named as test?

Reply:
its validation of templates written as tests so that we can make sure additions of new templates adhere to our restrictions.  This is written as tests so that PR's will fail the UT gates if the templates break our constraints.